content/articles/object-based-image-analysis-using-matlab/index.md

```diff
@@ -0,0 +1,241 @@
+
+### Introduction
+Object-based image analysis is the processing of an image based on the classification of its pixels to get more helpful information on the image. For example, such data can be based on height, edges, and boundaries. Matlab provides an interactive environment for object-based image analysis by executing functions used in analysis or inbuilt apps for image processing.
+
+Images contain objects with distinct regions. These regions have area, perimeter, shapes, and height features. Matlab allows for analysis of these properties using image analyzer functions or region props to obtain data from these images.
+
+Object-based image analysis is useful, especially in analyzing satellite maps, machine vision, and obtaining information based on object characteristics in an image.
```

Comment:
Generall, tell us what this article will cover. Kindly go thro' past articles to get more info about introductions!